src/CommunityToolkit.Maui.Core/Interfaces/IPopup.shared.cs

```diff
@@ -1,11 +1,15 @@
-﻿using IElement = Microsoft.Maui.IElement;
+using IElement = Microsoft.Maui.IElement;
 using LayoutAlignment = Microsoft.Maui.Primitives.LayoutAlignment;
 
 namespace CommunityToolkit.Maui.Core;
 
 /// <summary>
 /// Represents a small View that pops up at front the Page.
 /// </summary>
+#if NET10_0_OR_GREATER
+#error Remove IPopup
+#endif
+[Obsolete($"{nameof(IPopup)} is no longer used by {nameof(CommunityToolkit)}.{nameof(Maui)} and will be removed in .NET 10")]
```

Comment:
How about to add the attribute "editorBrowsable.never"? With that new users will not see it on intelissense, making sure they will use the new implementation

Reply:
Done!